waspc/data/Generator/templates/server/src/core/EntityValidationError.js

```diff
@@ -0,0 +1,22 @@
+class EntityValidationError extends Error {
```

Comment:
Yeah I think starting to segment under `core/error` makes sense, I can add that. Thanks

Reply:
Hey @Martinsos if we move `HttpError` under `core/error` instead of where it lives now, will that break existing apps who import like `import HttpError from '@wasp/core/HttpError.js'` today? If so, should we do that now, or in the future do you think? Thanks

Reply:
Uffff, good point @shayneczyzewski completely forgot about that! Yeah, I believe that would break them. In that case let's leave it as it is for now, and we will do this refactoring when we have a few more errors.